packages/lint-repository-sui/src/rules/node-version.js

```diff
@@ -0,0 +1,60 @@
+const dedent = require('string-dedent')
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Check that your repository use the latest Node version',
+      recommended: true,
+      url: null
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      moreThanOneNVMRC: dedent`
+        Your project has more than one .nvmrc file. That can be dangerous.
+        Please, use onle ONE in the root of your project.
+        If you are not sure about how do it, please contact with Platform Web.
+      `,
+      badNodeVersion: dedent`
+        Your current Node version is {{version}}.
+        Please be sure that your repository use the latest Node Version 20.
```

Comment:
Make sense if we set up the `Node Version 20` in a configuration file instead of hardcoded here?

Reply:
Yep, maybe we should save a constant containing the integer `20` because it's used more than once in this file.

Reply:
Make sense. I am going to change the lib to allow to use options.

Reply:
Cool, because I see some other potential options such as React version 😎